ruby-service.sql

```diff
@@ -0,0 +1,79 @@
+-- Drop previous versions of the tables if they they exist, in reverse order of foreign keys.
+
+DROP TABLE IF EXISTS Events;
+DROP TABLE IF EXISTS Trailer;
+DROP TABLE IF EXISTS Item;
+Drop TABLE IF EXISTS TrailerUser;
+DROP TABLE IF EXISTS TrailerItem;
+DROP TABLE IF EXISTS Users;
+
+-- Create the schema.
+CREATE TABLE Users (
+  username varchar(25) PRIMARY KEY NOT NULL,
+  password varchar(50) NOT NULL,
+	type varchar(10) NOT NULL
+	);
+
+CREATE TABLE Events (
+	ID integer PRIMARY KEY, 
+	name varchar(50),
+  time timestamp,
+  description text
+	);
+
+CREATE TABLE Trailer (
+  ID integer PRIMARY KEY, 
+	name varchar(25)
+	);
+
+CREATE TABLE Item (
+  ID integer PRIMARY KEY, 
+  name varchar(25),
+  quantity integer,
+  notificationlevel integer,
+  increment integer
+  );
+
+CREATE TABLE TrailerUser (
+  TID integer REFERENCES Trailer(ID),
+  UID varchar(25) REFERENCES Users(username)
+  );
+
+CREATE TABLE TrailerItem (
+  TID integer REFERENCES Trailer(ID),
```

Comment:
Based on the domain model each item belongs to a single trailer (which makes sense).  So it would work to just add a trailerID to the Item table

Reply:
Oops. Messed that one up.